### PR TITLE
Simplify the makefile setup to keep it focused on the basic tasks

### PR DESCRIPTION
**Pull request by danquah**

### CODE REVIEW

File: makeup-example/Makefile [hunk 1]
```diff
@@ -8,93 +8,65 @@
...
-#   make reset 	- stop all containers, update them and any dbs, and restart
-#   make test	- run any local tests
-#   make logs   - follow docker logs from the containers
-#   make status - show status of the environment
+#   make up 
+#   make stop
+#   make reset
+#   make test
+#   make logs
+#   make status
```
Comment (achton) [inline makeup-example/Makefile:17]: Should we include `make help` here then? Or simply refer people to `make help` for a full list of targets? It *is* self-documenting after all :-)

Comment (danquah) [reply makeup-example/Makefile:17]: done @achton

Comment (arnested) [reply makeup-example/Makefile:17]: I thought @achton meant only documenting `make help` and refer to that for getting the full list of targets and descriptions.

**Comment by arnested** [comment]
Status, @danquah?

**Comment by danquah** [comment]
@arnested we're _ever_ going to reach any agreement on this. How about merging this PR, then letting you get up and running with Balder, and then do a new PR with your adjustments? I'm good with most changes in particular when they come from a live project.

**Comment by arnested** [comment]
_Actually_ it seems this has been superseded by the work you did on https://github.com/reload/ddsdk2016/pull/299.

I based Balder on that work instead. I'll get around to doing a PR to proref for the Makefile as well.

**Comment by danquah** [comment]
Agreed

**Pull request closed without merging**